packages/dds/pact-map/src/interfaces.ts

```diff
@@ -12,7 +12,28 @@ export interface IPactMapEvents extends ISharedObjectEvents {
 	/**
 	 * Notifies when a new value goes pending or has been accepted.
 	 */
-	(event: "pending" | "accepted", listener: (key: string) => void);
```

Comment:
Now that we have an imperative way to get the details, probably omit the sequenceNumber from the event (folks can query for the details if they want them).  Basically same as how we don't include the new value on the event either.